apisix/plugins/opa/helper.lua

```diff
@@ -45,16 +52,136 @@ local function build_http_request(conf, ctx)
 end
 
 
-function _M.build_opa_input(conf, ctx, subsystem)
-    local request = build_http_request(conf, ctx)
+local function _build_http_route(conf, ctx)
+    local route_id = ctx.route_id
+    local routes = get_routes()
+
+    for _, route in ipairs(routes) do
+        if route.value.id == route_id then
+            return core.table.deepcopy(route).value
```

Comment:
Use table.clone is enough?

Reply:
Yes, deepcopy is too expensive to use and I have switched it to `table.clone`.